### DISCUSSION TITLE
Core team sync 2024-03-25

### DISCUSSION THREAD

**Discussion by madolson** [valkey-io/valkey#31] [General]
1. Discuss naming choices from the choices available in https://github.com/placeholderkv/placeholderkv/issues/9.
1.1. General consensus around the name ODIS being fairly unobjectionable, however it is similar to Redis. We're going to investigate further before committing to a decision.
2. Discuss initial version release strategy.
2.1. General consensus that we should release an initial version that is compatible with OSS Redis 7.2. In order to maximize compatibility with versioning scheme.
2.2. After initial release, we will release a new major version with more potentially breaking changes. Tentatively will call in the version 10 release, since it's how Microsoft counts.
2.3. No consensus about what types of major decisions should be made for the 7.2, but probably as few breaking changes as possible.

**Comment by kyle-yh-kim** [comment]
Personally, I second [OpenKV](https://github.com/placeholderkv/placeholderkv/issues/9#issuecomment-2015523796) from @hpatro. 

It rhymes well with other open-source projects such as OpenTofu and OpenSearch. If our goal is to move away from naming conventions similar "Redis", this would be our best option.
